plugins/com.google.cloud.tools.eclipse.appengine.whitelist.test/src/com/google/cloud/tools/eclipse/appengine/whitelist/XmlTest.java

```diff
@@ -50,4 +51,15 @@ public void testLoadExtensionClass() {
     }
   }
   
+  @Test
+  public void testLoadExtensionElements() {
+    NodeList extensions = document.getDocumentElement().getElementsByTagName("extension");
+    for (int i = 0; i < extensions.getLength(); i++) {
+      Element extension = (Element) extensions.item(i);
+      String point = extension.getAttribute("point");
+      Assert.assertTrue(point.length() > 0);
```

Comment:
There's no rule at Google about static imports, one way or the other, but personally I like to know wyhere my methods are coming from. :-)

Added the test that we can load extensions points.